### PR TITLE
Adding install instructions on README and listing Harpoon docs

### PR DESCRIPTION
**Pull request by hannelita**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,3 +4,218 @@
 about formal systems. It features direct support for object-level
 binding constructs using higher order abstract syntax and treats
 contexts as first class objects.
+
+
+## Installation & Configuration Guide
```
Comment (Ailrun) [inline README.md:9]: I remember @tsani said (I think similarly too) this Installation guide is too fragile to library change or so.
How about replacing this with the installation guide in readthedocs?

Comment (hannelita) [reply README.md:9]: I think we could have a small, summarized section for Install here at README; I'll send an update on that

Comment (Ailrun) [reply README.md:9]: Yeah, I agree with that.

File: README.md [hunk 2]
```diff
@@ -4,3 +4,218 @@
...
+$ .rvm/bin/rvm install 1.9.3
+```
+
+Note, you will want to add .rvm/bin to your .bashrc directory, so you will be
+able to find rvm without giving the full path.
+
+## Interactive Mode
+
+Beluga has a (legacy) interactive mode for Emacs. Check docs [here](https://github.com/Beluga-lang/Beluga/blob/master/interactive-mode-testing.md).
+We strongly advise using [Harpoon](https://github.com/Beluga-lang/Beluga/blob/master/harpoon.md). A detailed list of commands and tactics is listed [here](https://beluga-lang.readthedocs.io/en/latest/harpoon/index.html).
```
Comment (Ailrun) [inline README.md:202]: This `harpoon.md` is outdated. I think both link should point readthedocs, or change both the first and second link into something like
```
[Harpoon](https://beluga-lang.readthedocs.io/en/latest/harpoon/index.html)
[here](`https://beluga-lang.readthedocs.io/en/latest/harpoon/interactive-reference.html`)
```

File: README.md (hunk 1 above)
Comment (hannelita) [reply README.md:9]: Done

**Pull request merged**